Series expansion skill builders

gwynlovell opened the pull request:
# New content

<!--- Briefly describe the changes made. -->

<!--- Replace XXX with any associated issues. --->

Resolves #90 

## Checklist

- [x] New pages are added to the table of contents.
- [x] Everything displays correctly in the web-generated preview.
- [x] Appropriate attribution has been given to source materials.
- [x] My name is included alphabetically in `credits.md`.

mphoward reviewed (changes requested):
Thank you for your work on this! You did a great job typing everything up. I added a "solution" directive that will allow the reader to toggle the answer after they try it themselves. I also think it might be helpful to consolidate the solutions slightly. I applied these changes to the first two problems. Could you apply the same type of changes to the rest? Then, this should be good to merge!

wkdarko commented on calculus/series-expansion.md:
````diff
@@ -74,3 +74,179 @@ e^x &= \sum_{n=0}^{\infty} \frac{x^n}{n!} = 1 + x +  \frac{x^2}{2} + O(x^3) \\
+
+   ```{solution}
+   \begin{equation}
+   f(x)=cos(x) \approx 1-\frac{x^2}{2}
````
```suggestion
   f(x)=\cos(x) \approx 1-\frac{x^2}{2}
   ```

wkdarko commented on calculus/series-expansion.md:
```diff
@@ -74,3 +74,179 @@ e^x &= \sum_{n=0}^{\infty} \frac{x^n}{n!} = 1 + x +  \frac{x^2}{2} + O(x^3) \\
+   Recall:
+   
+   \begin{equation}
+   f(x)=cos(x) \approx 1-\frac{x^2}{2}
```
Let's keep the consistency so that these functions are upright rather than italic
```suggestion
   f(x)=\cos(x) \approx 1-\frac{x^2}{2}
```

wkdarko reviewed (commented):
Thanks for this nice work. I left a couple of minor suggestions but this is overall a great submission.

mphoward commented:
@gwynlovell please leave a comment here after you have addressed @wkdarko's feedback so I can give it a review myself!

gwynlovell commented:
> @gwynlovell please leave a comment here after you have addressed @wkdarko's feedback so I can give it a review myself!

I have addressed the feedback, thank you!

mphoward reviewed (approved):
Looks great, thanks very much!
